crates/solver/src/in_flight_orders.rs

```diff
@@ -89,6 +93,19 @@ impl InFlightOrders {
                 u256_to_big_uint(&order.data.buy_amount) > order.metadata.executed_buy_amount
             }
         });
+
+        tracing::debug!(
+            auction_block = %auction.block,
+            latest_settlement_block = %auction.latest_settlement_block,
+            inflight_before = %inflight_before.len(),
+            inflight_before = ?inflight_before,
+            inflight_after = %in_flight.len(),
+            inflight_after = ?in_flight,
```

Comment:
The keys `inflight_before` and `inflight_after` are duplicated with different values.

Reply:
This was intentional, I am not aware of any issues this would cause?

Reply:
Interesting. Didn't know that you can set the same key twice without overriding the value. 👌